src/core/labeling/qgspallabeling.h

```diff
@@ -798,6 +798,12 @@ class CORE_EXPORT QgsPalLayerSettings
      */
     QgsWkbTypes::GeometryType layerType = QgsWkbTypes::UnknownGeometry;
 
+    /**
+     * \brief string to show in the legend and in the preview icon
+     * \since QGIS 3.20
+     */
+    QString legendString = QObject::tr( "Aa" );
```

Comment:
Could you make this a full getter/setter instead of a public member variable? I've been moving to removing the public variables here as it's more flexible to have the proper getter/setter